src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -581,8 +568,10 @@ void NotationViewInputController::mouseMoveEvent(QMouseEvent* event)
     }
 
     // move canvas
-    m_view->moveCanvas(dragDelta.x(), dragDelta.y());
-    m_isCanvasDragged = true;
+    if (!m_view->isNoteEnterMode()) {
```

Comment:
for optimization: I think we can calculate the value of the note enter mode(see 534 line) and reuse it here

Reply:
fixed